fieldservice/report/fsm_order_report_template.xml

```diff
@@ -1,70 +1,65 @@
-<?xml version="1.0" encoding="utf-8"?>
 <odoo>
-  <template id="report_fsm_order_document">
-    <t t-call="web.external_layout">
-      <div class="page">
-        <div class="oe_structure"/>
-        <div class="row" name="location">
-          <div class="col-xs-5 col-xs-offset-1">
-              <strong t-if="doc.location_id == doc.customer_id">
-                Customer and Service Address:</strong>
-              <strong t-if="doc.location_id != doc.customer_id">
-                Service Address:</strong>
-              <div t-field="doc.location_id"
-                t-options='{"widget": "contact", 
-                            "fields": ["address", "name", "phone"]}'/>
-          </div>
-          <div class="col-xs-5 col-xs-offset-1">
-            <t t-if="doc.location_id != doc.customer_id">
-              <strong>Customer:</strong>
-              <div t-field="doc.customer_id"
-                t-options='{"widget": "contact",
-                            "fields": ["address", "name", "phone"]}'/>
+
+    <template id="report_fsm_order_document">
+        <t t-call="web.external_layout">
+            <t t-set="address">
+                <div t-field="doc.location_id.partner_id"
+                     t-options="{&quot;widget&quot;: &quot;contact&quot;, &quot;fields&quot;: [&quot;address&quot;, &quot;name&quot;, &quot;phone&quot;, &quot;email&quot;]}"/>
             </t>
-          </div>
-        </div>
-        <h2>
-          Field Service Order # <span t-field="doc.name"/>
-        </h2>
-        <h4>
-          <span t-field="doc.template_id.name"/>
-        </h4>
-        <div class="row mt32 mb32" id="informations">
-            <div t-if="doc.scheduled_date_start" class="col-xs-3">
-                <strong>Date Scheduled:</strong>
-                <p t-field="doc.scheduled_date_start"/>
-            </div>
-            <div t-if="doc.date_end and doc.state.is_closed" class="col-xs-3">
```

Comment:
I think it would be good to keep this section for Date Completed.  I often have to provide record of the service order to customers after the work is complete.